Classes/VS_MapList.uc

```diff
@@ -27,6 +27,9 @@ function int FindIndexForMap(string MapName) {
 			return Index;
 	}
 
+	if (Index < Maps.Length && Caps(Maps[Index]) < Caps(MapName))
+		Index += 1;
```

Comment:
This becomes unnecessary with the suggested change above.

Reply:
Will remove